paychmgr/store.go

```diff
@@ -45,8 +48,9 @@ const (
 )
 
 type VoucherInfo struct {
-	Voucher *paych.SignedVoucher
-	Proof   []byte
+	Voucher   *paych.SignedVoucher
+	Proof     []byte
+	Submitted bool
```

Comment:
This PR adds a field to `VoucherInfo`: `Submitted (bool)`
Will this cause any issues for people upgrading lotus, or does the datastore get reset?

Reply:
No, datastores won't get reset, looking at the generated cbor code, this will just be `false` for all vouchers

Which I _think_ is fine

Reply:
Ok as long as adding this field won't cause an error unmarshalling a `VoucherInfo` that was marshalled without the new field, then I think we're good

Reply:
Actually really we just need to make sure we can unmarshall an old format VoucherInfo into a new format VoucherInfo:
https://github.com/filecoin-project/lotus/pull/3321